manual_user_install.sh

```diff
@@ -0,0 +1,235 @@
+        systemctl --user enable ovos-audio
+        systemctl --user enable ovos-skills
+        systemctl --user enable ovos-phal
+        systemctl --user enable ovos-admin-phal
+        systemctl --user daemon-reload
+    fi
+    echo
+    echo "Done installing systemd files"
+    echo
+    }
+
+function install_extra_skills (){
+    echo
+    echo "Installing extra skills"
+    echo
+    # from NeonGeckoCom
+    pip3 install git+https://github.com/NeonGeckoCom/skill-user_settings
+    pip3 install git+https://github.com/NeonGeckoCom/skill-spelling
+    pip3 install git+https://github.com/NeonGeckoCom/skill-local_music
+    pip3 install git+https://github.com/NeonGeckoCom/skill-caffeinewiz
+
+    pip3 install git+https://github.com/OpenVoiceOS/skill-ovos-weather
+    pip3 install git+https://github.com/OpenVoiceOS/skill-ovos-hello-world
+
+    # common query
+    pip3 install git+https://github.com/OpenVoiceOS/skill-ovos-ddg
+    pip3 install git+https://github.com/OpenVoiceOS/skill-ovos-wolfie
+    pip3 install git+https://github.com/OpenVoiceOS/skill-ovos-wikipedia
+
+    # fallback
+    pip3 install git+https://github.com/OpenVoiceOS/skill-ovos-fallback-chatgpt
+
+    # OCP
+    pip3 install git+https://github.com/OpenVoiceOS/skill-ovos-news
+    pip3 install git+https://github.com/OpenVoiceOS/skill-ovos-somafm
+    pip3 install git+https://github.com/OpenVoiceOS/skill-ovos-youtube-music
+
+    # fun
+    pip3 install git+https://github.com/JarbasSkills/skill-icanhazdadjokes
+
+    # Here is where to include your local skills
+    if [[ ! -d $HOME/.local/share/mycroft/skills ]]; then
+        mkdir -p $HOME/.local/share/mycroft/skills
+    fi
+
+    cd $HOME
+    echo
+    echo "Done installing extra skills"
+    echo
+    }
+
+############################################################################################
+echo
+echo "This file will install ovos-core to this device"
+echo "using the latest commits from github."
+echo
+echo "First lets setup some things."
+echo
+read -p "Do you want to install systemd files (Y/n): " systemd
+if [[ -z "$systemd" || $systemd == y* || $systemd == Y* ]]; then
+    systemd="YES"
+    echo
+    read -p "Do you want to automatically start the ovos services? (Y/n): " enabled
+    if [[ -z "$enabled" || $enabled == y* || $enabled == Y* ]]; then
+        enabled="YES"
+    fi
+fi
+echo
+read -p "Would you like to install extra skills to match the downloadable image? (Y/n): " extra_skills
+if [[ -z "$extra_skills" || $extra_skills == y* || $extra_skills == Y* ]]; then
+    extra_skills="YES"
+fi
+echo
+echo "We are now ready to install OVOS"
+echo
+read -p "Type 'Y' to start install (any other key aborts): " install
+
+# update your system
+sudo apt update -y
+sudo apt upgrade -y
+
+if [[ $install == Y* || $install == y* ]]; then
+    if [[ ! -d $HOME/.local/bin ]]; then
+        mkdir -p $HOME/.local/bin
+    fi
+    export PATH=/home/ovos/.local/bin:$PATH
```

Comment:
Question: Is this something we should add to the bottom of the script to tell the user to add to their .profile / .bashrc file so they can access the binaries?

Reply:
It can probably go away. Your $PATH is adjusted automatically.

Reply:
Aha... I just looked into it.  The $PATH assignment is there to prevent a series of warning messages during the install. The export is a vestige of a previous experiment that has been deleted in my working copy.  

/home/ovos/.local/bin is added to your path after the first reboot following installation. Any way to automatically add that directory to the path short of rebooting?

Reply:
I think most scripts like this take the easy way out by just printing out instructions to the user. Something like:

"Please add this path to your PATH" or if you're feeling kind, give them the exact command they need to paste into their terminal